src/main/java/com/jagrosh/jmusicbot/commands/dj/RemoveRangeCmd.java

```diff
@@ -0,0 +1,107 @@
+package com.jagrosh.jmusicbot.commands.dj;

```

Comment:
License header should be present.
```suggestion
/*
 * Copyright 2021 John Grosh (jagrosh).
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *      http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
 
package com.jagrosh.jmusicbot.commands.dj;

```